src/main/java/org/generationcp/middleware/service/api/study/germplasm/source/GermplasmStudySourceInput.java

```diff
@@ -0,0 +1,50 @@
+package org.generationcp.middleware.service.api.study.germplasm.source;
+
+import org.generationcp.middleware.pojos.GermplasmStudySourceType;
+
+public class GermplasmStudySourceInput {
```

Comment:
add equals, toString and hashCode in case we need to create sets

Reply:
Done